### ISSUE TITLE
centos:7 does not include the 'which' command

### ISSUE THREAD

**Issue by dmd** [ReproNim/neurodocker#317]
`docker run --rm kaczmarj/neurodocker:v0.3.2 generate --base centos:7 --pkg-manager yum --fsl version=6.0.1`

fails with:

```
Failed to download Miniconda - see /tmp/fslpython86Ug/fslpython_miniconda_installer.log for details
```

That file contains:

```
Installing FSL conda distribution into /opt/fsl/fslpython
/opt/fsl/etc/fslconf/fslpython_install.sh: line 141: which: command not found
/opt/fsl/etc/fslconf/fslpython_install.sh: line 100: -O: command not found
```

because `centos:7` is extremely minimal and does not include the package `which` (no, it's not a bash builtin!)

**Comment by gdevenyi** [comment]
Well FSL should be using "command -v" which is POSIX and portable...

https://pubs.opengroup.org/onlinepubs/009695399/utilities/command.html

But its a bit surprising that which isn't installed.

**Comment by dmd** [comment]
```
$ docker run -it centos:7 bash
[root@97278c3242d3 /]# which yes
bash: which: command not found
[root@97278c3242d3 /]# command -v yes
/usr/bin/yes
```

**Comment by kaczmarj** [comment]
@dmd - thanks for the report. i responded in https://github.com/ReproNim/neurodocker/issues/318 with a similar request. can you please retry with the latest version of neurodocker? version 0.6.0.

that of course won't change the fact that centos 7 does not have which by default, but the fsl installation procedure was changed.

**Comment by djarecka** [comment]
looks like the issue was addressed in #318
